content/en/docs/How to/enable-autocomplete.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: "Enable autocomplete for shell"
-linkTitle: "Enable autocomplete"
+linkTitle: "Autocompletion"
 weight: 10
 description: >
-  Enables autocomplete for the Score implementation (CLI).
+  How to autocomplete for the Score implementation (CLI)
+Alias: 
+- /docs/reference/ide-linter-autocomplete/
```

Comment:
Changed it to `/docs/reference/enable-autocomplete` - LMK if that works

Reply:
As it is, it does not work technically because the `Alias:` statement needs to be `aliases:`. With that modification, it does.

The path `/docs/reference/enable-autocomplete` does not exist in the documentation that's presently live, so I guess we do not need this alias entry at all. If I'm not missing something, and there's no other reason to have it, let's just remove it altogether 🪓

Reply:
@TobiasBabin you're right - the page that currently exists is https://docs.score.dev/docs/reference/score-cli/enable-autocomplete/, sorry i've gotten all confused working with different versions of the docs. I guess it would make sense to add it as an alias? Otherwise happy to remove it fully

Reply:
My bad, I missed the existing page you now mentioned. In that case, we should be all means preserve `/docs/reference/score-cli/enable-autocomplete/` as a link target. I added this alias as a commit suggestion.